src/lib/fetchers/ac/fetchApp.ts

```diff
@@ -0,0 +1,21 @@
+import { IApp } from '../../model/ac/App';
+import { fetchDocumentWithTypeTag } from './fetchDocument';
+
+export function interpretJSONResponseAsApp(json: unknown): IApp {
+  return json as IApp
+}
+
+export function appIdForHostname(hostname: string) {
+  const cleanedHostName = hostname
+    // Allow sharing a deployment with ngrok
+    .replace(/.*\.ngrok(?:-free)\.app$/, 'wheelmap.tech')
+    // Allow branch test deployments
+    .replace(/.*\.wheelmap\.tech$/, 'wheelmap.tech')
+    // Use 'localhost' as hostname for loopback IP
+    .replace(/127\.0\.0\.1/, 'localhost')
```

Comment:
This is a very, very minor thing for an ancient edge case, but some versions of Windows don't resolve `localhost` as `127.0.0.1` (something like Win 7/8 home or so). If the benefits of your setup outweigh it, just keep it like that.